bpf/init.sh

```diff
@@ -413,17 +424,15 @@ else
 	fi
 fi
 
-case "${MODE}" in
-	"ipvlan")
-		;;
-	*)
-		# bpf_host.o requires to see an updated node_config.h which includes ENCAP_IFINDEX
-		CALLS_MAP="cilium_calls_netdev_ns_${ID_HOST}"
-		POLICY_MAP="cilium_policy_reserved_${ID_HOST}"
-		OPTS="-DFROM_HOST -DFIXED_SRC_SECCTX=${ID_HOST} -DSECLABEL=${ID_HOST} -DPOLICY_MAP=${POLICY_MAP}"
-		bpf_load $HOST_DEV1 "$OPTS" "egress" bpf_netdev.c bpf_host.o from-netdev $CALLS_MAP
-		;;
-esac
+# bpf_host.o requires to see an updated node_config.h which includes ENCAP_IFINDEX
+CALLS_MAP="cilium_calls_netdev_ns_${ID_HOST}"
+POLICY_MAP="cilium_policy_reserved_${ID_HOST}"
+if [ "$MODE" == "ipvlan" ]; then
+	OPTS="-DENABLE_EXTRA_HOST_DEV=1 -DFROM_HOST -DFIXED_SRC_SECCTX=${ID_HOST} -DSECLABEL=${ID_HOST} -DPOLICY_MAP=${POLICY_MAP}"
```

Comment:
Minor tidy, could this instead be more like

```
OPTS="....default set"
if [ "$MODE" == "ipvlan" ]; then
  OPTS += "ipvlan-specific opts"
fi
```

Reply:
Agree, much better, will do.